<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckOut</class>
 <widget class="QMainWindow" name="CheckOut">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckOut</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OkButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="total">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Total :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="total_lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="percentage_discount_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>276</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imgs.qrc">:/new/prefix1/Pheonix.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="price_lineEdit">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>220</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="percentage_discount">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>276</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Voucher</string>
    </property>
   </widget>
   <widget class="QLabel" name="quantity">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>248</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QLabel" name="Price">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>220</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Price </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="percentage_discount_lineEdit">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>276</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantity_lineEdit">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>250</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="drug_name">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>193</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Drug name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="drug_name_lineEdit">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>193</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Checkout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Checkout">
   <property name="text">
    <string>About Checkout</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
